src/tool/crosshair.js

```diff
@@ -56,6 +56,7 @@ export default function() {
                 .style('pointer-events', 'all')
                 .on('mouseenter.crosshair', mouseenter)
                 .on('mousemove.crosshair', mousemove)
+                .on('wheel.crosshair', mousewheel)
```

Comment:
since it's working well with the same code, looks like they can be the same function. Just pushed a commit which eliminates the separate mousewheel function